noah/src/main/resources/crawler/health_map_crawler.py

```diff
@@ -0,0 +1,116 @@
+import urllib2
+import urllib
+import datetime
+import io
+import os
+import json
+import sys
+
+def argumentCheck():
+    if len(sys.argv) != 4:
+        sys.exit("Invalid number of Arguments... \n Enter the directory name, filename where json data needs to be stored and number of days in arguments: \n Example: python health_map_crawler.py HealthMapResult 365")  
```

Comment:
the example is wrong? 
`python health_map_crawler.py HealthMapResult 365` only has 2 arguments.